x-pack/metricbeat/module/meraki/_meta/config.yml

```diff
@@ -0,0 +1,6 @@
+- module: meraki
+  metricsets: ["device_health"]
+  enabled: true
+  period: 300s
```

Comment:
Is a 5 minutes period sufficient for Meraki ?
What is the refresh interval internally for metrics of Meraki service ?

Reply:
are you concerned about this being too long or too short?

i see uplink metrics in one minute buckets - the other metrics are hard to test because we don't have them in our test environment. but they appear default to 1 day (or longer) time periods in 10 minute buckets. there doesn't appear to be a limitation in how short the time period is for these metrics.

Reply:
I was concerned about it being long. 
For uplink, do you think, we should have shorter period ?

Reply:
yeh, this is just a default though, don't need to worry about it now. you still get all the uplink metrics for the previous 5 minutes reported. so in the end you have one data point per minute. any longer than 5 minutes and you start to miss data points, which is why i included the upper bound on the period when the module starts up.